### DISCUSSION TITLE
Temporal plugin configuration for database connection

### DISCUSSION THREAD

**Discussion by suityou01** [roadrunner-server/roadrunner#1891] [Q&A] [answered]
I would like to use Temporal with Roadrunner and set up a SQL database (mysql) for the data store as described [here](https://docs.temporal.io/references/configuration#sql)

Is this possible to set up with Roadrunner? Where would this configuration go? In the main rr.yaml or in the worker.php? Somewhere else?

Thanks in advance.

**Comment by rustatian** [comment]
Hey @suityou01 👋 
Definitely, but this configuration is for the Temporal cluster and Temporal server. RR connects to the Temporal cluster, but configuration is up to you.

**Reply by rustatian** [reply, marked as answer]
I mean, that the cluster configuration with DB and other things specified by the link you've sent is usually a dev-ops task. If you need some local env, you may use docker: https://github.com/temporalio/roadrunner-temporal/blob/master/tests/env/docker-compose-temporal.yaml

**Reply by suityou01** [reply]
Wonderful, thank you. I guess I was having trouble understanding if it was just the Temporal Client that ships with Roadrunner or the whole stack. As my load is relatively low I may well install both in my container to it is a self contained unit.

**Reply by rustatian** [reply]
You're welcome 😃
